### PR TITLE
Add permissions for omnipotence rights, Add static folder for new pac…

### PR DESCRIPTION
**Pull request by gauransh7**
- Add permissions for omnipotence rights
[Used here in filemanager](https://github.com/IMGIITRoorkee/omniport-django-filemanager/blob/filemanager_revamp/views.py)

### CODE REVIEW

**Review by dhruvkb — CHANGES_REQUESTED** [review]
Why does this PR contain JS files for a carousel?

**Comment by gauransh7** [comment]
> Why does this PR contain JS files for a carousel?

These files came along with a package(pure-react-carousel). Only one of the four files was of use, so I have made a commit to remove the needless files.

**Comment by dhruvkb** [comment]
Even if it's useful, two separate unrelated features should belong in two separate PRs.

**Comment by gauransh7** [comment]
> Even if it's useful, two separate unrelated features should belong in two separate PRs.

Yes Sure.
Made a different PR for package changes.
Ref.- [https://github.com/IMGIITRoorkee/omniport-backend/pull/140](https://github.com/IMGIITRoorkee/omniport-backend/pull/140)

File: omniport/core/kernel/permissions/omnipotence.py [hunk 1]
```diff
@@ -1,5 +1,7 @@
 from django.conf import settings
```
Comment (pradumangoyal) [inline omniport/core/kernel/permissions/omnipotence.py:2]: This line is not needed

Comment (gauransh7) [reply omniport/core/kernel/permissions/omnipotence.py]: yes sure.

**Pull request merged**